api/PositionOptions.json

```diff
@@ -116,13 +63,61 @@
           "deprecated": false
         }
       },
+      "secure_context_required": {
+        "__compat": {
+          "description": "Secure context required",
+          "support": {
+            "chrome": {
+              "version_added": "47"
+            },
+            "chrome_android": {
+              "version_added": "47"
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": "55"
+            },
+            "firefox_android": {
+              "version_added": "55"
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": true
+            },
+            "opera_android": {
+              "version_added": true
+            },
+            "safari": {
+              "version_added": true
+            },
+            "safari_ios": {
+              "version_added": true
+            },
+            "samsunginternet_android": {
+              "version_added": null
+            },
+            "webview_android": {
+              "version_added": false
```

Comment:
Seems like more than just a rearranging of the data happened here. Is there more to the story behind version "47" going away for WebView?

Reply:
I used the information from the [Legacy compatibility table](https://developer.mozilla.org/docs/Web/API/PositionOptions#Browser_compatibility) and cross-referenced this with the information in [`Geolocation`](https://developer.mozilla.org/docs/Web/API/Geolocation#Browser_compatibility).